### PR TITLE
[Xcode14] Bump xcode 14 to beta 3.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tests/xtro-sharpie/Makefile [hunk 1]
```diff
@@ -294,3 +294,5 @@ unclassified2todo: $(U2TODO)
 unclassified2todo-dotnet: $(U2TODO)
 	$(SYSTEM_CSC) -features:strict Filter.cs u2todo/u2todo.cs
 	cd $(DOTNET_ANNOTATIONS_DIR) && $(MONO) $(abspath $(U2TODO))
+
+unclassified2todo: unclassified2todo-classic unclassified2todo-dotnet
```
Comment (mandel-macaque) [inline tests/xtro-sharpie/Makefile:298]: @rolfbjarne I am lazy and have bad memory.

File: tests/xtro-sharpie/api-annotations-dotnet/common-Foundation.ignore [hunk 2]
```diff
@@ -790,7 +790,6 @@
 !unknown-type! NSObject bound
 
 ## Used 'INSFastEnumeration' placeholer (just so we can start flagging things)
-!missing-protocol-conformance! NSDictionary should conform to NSFastEnumeration (defined in 'NSGenericFastEnumeraiton' category)
```
Comment (rolfbjarne) [inline tests/xtro-sharpie/api-annotations-dotnet/common-Foundation.ignore:793]: This has been removed here and added to each platform-specific file, which is not necessary.

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: tests/xtro-sharpie/api-annotations-dotnet/common-Foundation.ignore (hunk 2 above)
Comment (mandel-macaque) [reply tests/xtro-sharpie/api-annotations-dotnet/common-Foundation.ignore:793]: That was the tool, we might need to fix it.

File: tests/xtro-sharpie/common-Foundation.ignore [hunk 3]
```diff
@@ -802,7 +803,6 @@
 !unknown-type! NSObject bound
 
 ## Used 'INSFastEnumeration' placeholer (just so we can start flagging things)
```
Comment (rolfbjarne) [inline tests/xtro-sharpie/common-Foundation.ignore:805]: 1. The same change is needed in the api-annotations-dotnet subdirectory.
2. You moved the line in this file in the end, so there's a dangling comment here now.

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: tests/xtro-sharpie/common-Foundation.ignore (hunk 3 above)
Comment (mandel-macaque) [reply tests/xtro-sharpie/common-Foundation.ignore:805]: Funny thing, locally it did not complain about the api-annotations-dotnet

**Comment by mandel-macaque** [comment]
mtouch tests are known: https://github.com/xamarin/xamarin-macios/pull/15422

**Comment by mandel-macaque** [comment]
landing, since we created https://github.com/xamarin/maccore/issues/2602

**Pull request merged**